tests/unit/execution_tree/primitives/inverse_operation.cpp

```diff
@@ -29,7 +29,7 @@ void test_transpose_operation_0d()
         transpose.eval();
```

Comment:
Why's the inverse test actually testing transpose?

Reply:
@hkaiser :)))) Who wrote this test? It does test transpose and functionality is correct but most of the method and variable names are wrong

Reply:
Renamed in 42b1a0af2076deefdf48d22333442bc92de4045f

Reply:
Thanks!